doc/advanced_topics.rst

```diff
@@ -0,0 +1,331 @@
+###############
+Advanced Topics
+###############
+
+In this section, we illustrate some topics which experienced
+users are expected to use in HPC workloads, especially when
+simulation and AI are required to interact. The topics are
+explained through code snippets,
+with code which goes beyond SmartSim and SmartRedis API
+(e.g. code showing how to jit-script a PyTorch model): the
+intention is that of showing *one* simple way of leveraging
```

Comment:
the phrasing after the ":" seems a little unclear.  Did we mean something like "the intention is to show *one* simple way of leveraging a feature while acknowledging other optimized SmartSim syntax may be applicable to specific use cases."

Reply:
It should be clearer now.